Update how-to-enable-dmarc-reporting-for-microsoft-online-email-routi… (#12286)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/step-by-step-guides/how-to-enable-dmarc-reporting-for-microsoft-online-email-routing-address-moera-and-parked-domains.md

1. Select **+ Add record**.
1. A flyout will appear on the right. Ensure that the selected Type is **TXT (Text)**.
1. Add `_dmarc` as **TXT name**.

> 1. Add your specific DMARC value. For more information check [how to Form the DMARC TXT record for your domain](/microsoft-365/security/office-365-security/email-authentication-dmarc-configure#step-4-form-the-dmarc-txt-record-for-your-domain)

Reviewer: ```suggestion
1. Add your specific DMARC value. For more information, see [Form the DMARC TXT record for your domain](/microsoft-365/security/office-365-security/email-authentication-dmarc-configure#step-4-form-the-dmarc-txt-record-for-your-domain).
```
